### PR TITLE
Added missing have_rocksdb.inc

### PR DESCRIPTION
**Pull request by george-lorch**
Upstream commit ID: 813cb8923aae97773803833d89dbc04c9de21669

### CODE REVIEW

**Review by laurynas-biveinis — COMMENTED** [review]
Why not convert to plain have_rocksdb.inc at the same time?

File: mysql-test/suite/rocksdb.sys_vars/t/all_vars.test [hunk 1]
```diff
@@ -1,3 +1,4 @@
+--source include/have_rocksdb_as_default.inc
```
Comment (laurynas-biveinis) [inline mysql-test/suite/rocksdb.sys_vars/t/all_vars.test:1]: This testcase only needs MyISAM, not RocksDB :)

Comment (george-lorch) [reply mysql-test/suite/rocksdb.sys_vars/t/all_vars.test:1]: I was kind of thinking the same thing, it _does_ need RocksDB for the vars to be present in the first place to 'see' the vars, but no, it does not invoke RocksDB for the purposes of executing anything against it.

**Comment by george-lorch** [comment]
Not converting at the same time just to keep scope of the commit down as some of the tests may actually need fixup and re-recording. Also, this was effectively a merge in of a (proposed) upstream commit. Converting tests will come soon enough, I want to get the ones that are converted either passing or disabled until their features are properly ported/implemented.

**Pull request merged**